uk_bin_collection/uk_bin_collection/councils/LancasterCityCouncil.py

```
@@ -57,30 +70,48 @@ def parse_data(self, page: str, **kwargs) -> dict:
+                            None,
+                        )
+                        if collection_type is None:
+                            collection_type = type_text
+                            for suffix in SUFFIXES:
+                                collection_type = collection_type.removesuffix(suffix)
+                        if collection_type is not None:
                             data["bins"].append(
                                 {
-                                    "type": bin_type_element.text.strip().removesuffix(" Collection Service"),
+                                    "type": collection_type,
                                     "collectionDate": dt.strftime(date_format),
```

Ruff (0.15.6)

[error] `date_format` may be undefined, or defined from star imports

(F405)